### ISSUE TITLE
Cleanup stale branches

### ISSUE THREAD

**Issue by Michiel-s** [AmpersandTarski/Ampersand#1038]
Hi @stefjoosten,

I noticed a couple of stale branches initiated by you. Can you help clean this up.
For a couple of them I think we should simply delete them, because due to the many changes made later we can't merge those anyway.

What to do with:
- [ ] feature/Archimate3 (latest commit 20 sept). Including work on a Dockerfile, which is completely refactored now. Is the branche 'Archimate_Extension' a replacement?
- [ ] feature/Archimate_extension (latest commit 2 aug).
- [ ] Archimate_Extension, are we going finish this feature sometime? Branch originates from commit (4496c8fba 17 mei 2016!). Contains more merge commits from development that actual work done on this feature.

- [x] feature/Atlas (latest commit 14 dec)
- [x] feature/Decouple-GraphViz (latest commit 11 sept).
- [x] feature/unifiedInterfaces (only 2 commits from 2016!) by @sjcjoosten 
- [x] data-analysis (empty branch, i.e. no commits other that a merge from development)

**Comment by sjcjoosten** [comment]
Let's remove the unifiedInterfaces, it's properly stale

**Comment by Michiel-s** [comment]
Related to #1027

**Comment by stefjoosten** [comment]
- I have deleted `feature/unifiedInterfaces` on gitHub (aka origin) based on Sebastiaan's comment above.
 - I have deleted `feature/Atlas` too.
 - I will delete all Archimate branches as soon as I have verified that the Archimate functionality works in branch fixAtlasComplete. I know @hanjoosten has made the effort to put it into mainstream Ampersand in mentioned branch.
 - I have deleted data analysis. I saved an instruction for git to handle Excel-files properly and committed it to development.
 - I have deleted `feature/Decouple-GraphViz`
